Python/ceval_gil.c

```diff
@@ -1319,5 +1319,70 @@ _Py_HandlePending(PyThreadState *tstate)
             return -1;
         }
     }
+
+#ifdef Py_REMOTE_DEBUG
+    const PyConfig *config = _PyInterpreterState_GetConfig(tstate->interp);
+    if (config->remote_debug) {
+        if (tstate->remote_debugger_support.debugger_pending_call) {
```

Comment:
Likewise we said in the PEP that we will only be accepting 0 or 1 for `debugger_pending_call` for now.
```suggestion
        if (tstate->remote_debugger_support.debugger_pending_call == 1) {
```